models/audit_opinions.json

```diff
@@ -1,7 +1,7 @@
 {
     "fact_table": "audit_opinion_facts",
     "label": "Audit Opinions",
-    "description": "",
+    "description": "Opinions of financial performance from the Auditor General.",
```

Comment:
isn't it opinions of audit result? I haven't read these but I didn't think they look at financial performance (in the sense we're looking at performance), but primarily "Are their finances recorded as they should be?"